In [ ]:
#import the libraries
import pandas as pd
import scipy
import numpy as np
from scipy import stats

**HYPOTHESIS TESTING**

Given data is sample, we should calculate by t-distribution.

For given problem, h0: There is no difference in cost model, it will be accurate
                   h1: There is a difference in cost model, it will can be inaccurate.



By t-distribution we can calculate,

In [27]:
mean = 1000 + 5 * 600
sample_mean = 3050
standard_deviation = 5*25
sample_size = 25
alpha = 0.05

In [28]:
standard_error = standard_deviation / np.sqrt(sample_size)
t_statistic = (sample_mean - mean) / standard_error
print('t statistic:',t_statistic)
print('standard_error',standard_error)

t statistic: -38.0
standard_error 25.0


In [30]:
from scipy import stats
degrees_of_freedom = sample_size - 1
critical_value = stats.t.ppf(1.0 - alpha, degrees_of_freedom)
print('critical value:',critical_value)

critical value: 1.7108820799094275


By above t-distribution, we have got t_statistic = -38.0 and critical value = 1.71

t_statistic < critical value.

This means we reject the h0.

The cost model is inaccurate, the opening costs are significantly higher than the model predicts.